Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift

```diff
@@ -47,7 +47,7 @@ struct AnimationsState: Equatable {
 
 enum AnimationsAction: Equatable {
   case circleScaleToggleChanged(Bool)
-  case dismissAlert
+  case alertDismissed
```

Comment:
Can you please alphabetize this so it comes first now?

Reply:
Should I change it "dismissAlert" back? but in `concurrency-updates-proto` branch, "dismissAlert" case is found in only 01-GettingStarted-Animations.swift. Can you tell me why?

Reply:
I wouldn't worry about any experimental branches past `concurrency-updates` for now :) Some of those future branches are able to simplify cases away, which may explain that omission, but I need to refresh my memory to understand why that particular case is missing.

Reply:
Got it. I reverted this case in https://github.com/pointfreeco/swift-composable-architecture/pull/1240/commits/be713f8f2d284d4c4ee9c6daff9c17490b6b29b1 commit. Please review again.
I'm expecting new concurrency features! Thank you for maintaining this well designed library. 👍🏻